examples/cra-kitchen-sink/package.json

```diff
@@ -22,14 +22,15 @@
     "@storybook/addon-actions": "^3.0.0",
     "@storybook/addon-centered": "^3.0.0",
     "@storybook/addon-events": "^3.0.0",
-    "@storybook/addon-knobs": "^3.0.0",
     "@storybook/addon-info": "^3.0.0",
+    "@storybook/addon-knobs": "^3.0.0",
     "@storybook/addon-links": "^3.0.0",
     "@storybook/addon-notes": "^3.0.0",
     "@storybook/addon-options": "^3.0.0",
     "@storybook/addon-storyshots": "^3.0.0",
     "@storybook/addons": "^3.0.0",
     "@storybook/react": "^3.0.0",
+    "babel-plugin-react-docgen": "^1.5.0",
```

Comment:
This is included by the `package.json` in the react app already and it should be used by the babel there already.

https://github.com/storybooks/storybook/blob/master/app/react/package.json#L34

If it's not, let's fix it.